README.md

```diff
@@ -20,6 +20,17 @@ echo "sdk.dir="$ANDROID_HOME > local.properties
 ./gradlew assembleDebug
 ```
 
+### How to sent APK to device
+
+The following steps are only needed if running via command line. They are not needed if you are building via Android Studio.
+
+Ensure that that ADB recognizes your device with `$ adb devices`.
+
+If a single device exists, install via `$ adb install /path/to/apk.apk`.
+
+Assuming you used Gradle like earlier, the file will be called `module_name-debug.apk` in `project_name/module_name/build/outputs/apk/`.
+
+Alternatively, you can simply run `./gradlew installDebug` to build, deploy, and debug all in a single command.
```

Comment:
Thanks for your PR @Nefari0uss!

What do you think about how to run that command on the Windows machines? 
We could change it to:

```
Alternatively, you can simply run `./gradlew installDebug` (or `gradlew installDebug ` on Windows OS) to build, deploy, and debug all in a single command.
```

or

```
Alternatively, you can simply run `./gradlew installDebug` to build, deploy, and debug all in a single command.
Note: Run  `gradlew installDebug` on Windows machine.
```
Feel free to change it to cover Windows users. 
[Info](https://developer.android.com/studio/build/building-cmdline.html).

Reply:
Sure. I can change that once I get home from work.

Reply:
@Nefari0uss Thanks for your contribution! I spot a typo on line 23, where it says **sent** should be **send**